### DISCUSSION TITLE
How to customize dropdown list of VAutocomplete for multiple searches.

### DISCUSSION THREAD

**Discussion by CodeKnight-max** [vuetifyjs/vuetify#20031] [Q&A] [answered]
Hi everybody,
Hope this helps to make a creative Veutify component.
I need a custom autocomplete component, which has several lists as its dropdown.
So, it may look like the following:
![image](https://github.com/vuetifyjs/vuetify/assets/144061915/30ea0484-5e3a-40e1-ba73-8a1277db243b)
This will result in several kinds of lists being filtered.

**Comment by YSHgroup** [comment, marked as answer]
Hi @CodeKnight-max ,
As a developer using Vue and Vuetify a lot, it sounds very interesting. So, I just tried to resolve your problem and hope this help you and others who would face with such ones.

I've found a solution for your requirements by customizing the `item` slot in VAutocomplete component.
If you need it, please refer [my code lines](https://github.com/YSHgroup/turborepo-best/blob/main/apps/vuetify-custom/src/views/VuetifyComponents/MultiSearchField.vue).
![image](https://github.com/vuetifyjs/vuetify/assets/146746648/b25678a5-bb6c-4fe6-8df0-d3204bbee1b6)

The item slot exists to customize the individual items in a dropdown, but here that is utilized to make multi list in a dropdown.
And then search and select functionality are implemented separately.

Hope there are other best solution else.

**Reply by CodeKnight-max** [reply]
Thanks for your answer. It works correctly on my side too.
